src/main/resources/application.yaml

```diff
@@ -7,15 +7,17 @@ management:
 
 idam:
   s2s-auth:
-    url: ${S2S_URL}
+    useStub: false
+    url: ${S2S_URL:-}
```

Comment:
Do we need `-` as default value?

Reply:
The functional tests complained at me for some reason. I didn't think it should need it either. This is another one where hopefully I can remove it once testing out @doncem idea